Bottom: a61efc836c044069888cf21d104849f3947efc2d
Top:    8954b93c4a24f7eff5fe1fbdab2e5447284379eb
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-02 19:48:56 -0500

Refresh of jds-lirc

---

diff --git a/drivers/input/ir-configfs.c b/drivers/input/ir-configfs.c
index ac0326d..90fd806 100644
--- a/drivers/input/ir-configfs.c
+++ b/drivers/input/ir-configfs.c
@@ -125,6 +125,8 @@ static void keymap_release(struct config_item *item)
 {
 	struct keymap *keymap = to_keymap(item);
 	struct remote *remote = to_remote(to_config_group(item->ci_parent));
+
+	printk("keymap release\n");
 	clear_bit(keymap->keycode, remote->input->keybit);
 	kfree(keymap);
 }
